app/src/main/java/com/okta/oidc/example/PlainActivity.java

```diff
@@ -403,11 +406,22 @@ public void onError(String error,
 
     @Override
     protected void onActivityResult(int requestCode, int resultCode, Intent data) {
-        if (requestCode == REQUEST_CODE_CREDENTIALS && resultCode == RESULT_OK) {
-            if (mCurrentEncryptionManager.getCipher() == null) {
-                mCurrentEncryptionManager.recreateCipher();
+        if (requestCode == REQUEST_CODE_CREDENTIALS) {
```

Comment:
nit: The `if`s are piling up here. Would be easier to read and neater if we used guard statements instead.

Reply:
Will clean up in next demo pass.